uvm/hier/usr/lib/python3/dist-packages/tests/test_network.py

```diff
@@ -2368,6 +2368,22 @@ def test_606_troubleshooting_trace(self):
         # Don't care about success/failure just that we see the test ran
         assert "tcpdump:" in output, "trace test"
 
+    def test_701_remove_dpkg_files(self):
+        """
+        Removing dpkg files
+        """
+        dpkg_file = '/etc/untangle/iptables-rules.d/740-test.dpkg-old'
```

Comment:
Also add similar files to other directories under /etc/untangle and verify they're all removed.

Reply:
Added logic to add similar files for directories under /etc/untangle, and verified they are removed.